conf/modules/short_variant_calling.config

```diff
@@ -29,7 +29,8 @@ process {
         ext.args = { [
             "--sample_name=${meta.id}",
             "--model_type=${params.deepvariant_model_type}",
-            '-vcf_stats_report=False'
+            '-vcf_stats_report=False',
+            meta.sex == 1 ? '--haploid_contigs chrX,chrY' : ''
```

Comment:
```suggestion
            meta.sex == 1 ? '--haploid_contigs="chrX,chrY"' : ''
```
most of the other DV options seems to have this format and it is the format used in their doc, https://github.com/google/deepvariant/blob/r1.6.1/docs/deepvariant-haploid-support.md